graphistry/embed_utils.py

```diff
@@ -301,12 +301,14 @@ def embed(
         """
         # this is temporary, will be fixed in future releases
         try:
-            if isinstance(self._nodes, cudf.DataFrame):
+            # if isinstance(self._nodes, cudf.DataFrame):
```

Comment:
remove commented lines

Reply:
probably better as lazy import and then the whoe `cudf is not None and isinstance(self._nodes, cudf)` (make a helper, `lazy_isinstance` ?)

Reply:
same for all the below too

Reply:
lazy import of cudf (`check_cudf`) was slowing import of graphistry. in embed_utils we dont make cudf df , just check for them... and for some reason send them all to_pandas (need to ask tanmoy about this
)